src/setup/devnet/index.js

```diff
@@ -255,6 +255,18 @@ export class Devnet {
             this.config.targetDirectory
           )
 
+          // TODO: Uncomment when finalised for docker setup
```

Comment:
Did we even tested the docker setup with this code uncommented? In case we did not, can you create a task in JIRA? Thanks

Reply:
Sure, will do. Thanks!
This was more of a reminder to check for docker setup once remote is finalised.